src/main/definitions/dates.ts

```diff
@@ -96,5 +105,9 @@ export const StartDateFormFields = {
   hint: (l: AnyRecord): string => l.hint,
   values: DateValues,
   validator: (value: CaseDate): DateTypes =>
-    areDateFieldsFilledIn(value) || isDateInputInvalid(value) || isFutureDate(value),
+    areDateFieldsFilledIn(value) || isDateInputInvalid(value) || isDateInPast(value),
+};
+
+export const convertDateToCaseDate = (date: Date): CaseDate => {
+  return { day: `${date.getDate()}`, month: `${date.getMonth() + 1}`, year: `${date.getFullYear()}` };
```

Comment:
these don't need to be in template literals

Reply:
I needed to convert them to strings anyway (that's how CaseData stores it), is there a nicer way to do it?